content/faq/referrals.md

```diff
@@ -3,27 +3,27 @@ title: How does the referral program work?
 
 *Note: Referral redemptions are currently limited to 1*
 
-### How many credits do I get and how do I see how many credits I've earned?
+### How many LBRY Credits do I earn and how do I see how many credits I've earned?
 
 New invites sent via the LBRY app are eligible for a 3 LBC reward amount. 
 
-For invites prior to 8/1/2017, the current referral reward, as well as the number of people you have referred, has been recorded by our system. You can see the list of your invited users on the Invite History section of the Invites screen.
+For invites prior to 8/1/2017, the current referral reward, as well as the number of people you have referred, has been recorded by our system. You can see the list of your invited users on the "Invite History" section of the Invites screen.
 
-You will always receive the number of credits being awarded at the time your referrer joined the list. Currently, the reward amount will be shown when the invite is claimed. You can see the reward amount in the Rewards section of the LBRY app after it is claimed. 
+You will always receive the same number of credits being awarded at the time your referrer joined the list. Currently, the reward amount will be shown when the invite is claimed. You can see the reward amount in the "Rewards" section of the LBRY app after it is claimed. 
 
 ### When do I get my rewards?
 
-We are currently in a testing phase of referral system where each account is limited to a single referral reward redemption.  
+We are currently in a testing phase of the referral system where each account is limited to a single referral reward redemption.  
 
-You will not receive rewards for each referral until those users join and use LBRY themselves (see below section regarding claimable referrals). This is to combat referral fraud.
+You will not receive rewards for each referral until those users join and use LBRY themselves (see below section regarding claimable referrals). This is to combat referral fraud and abuse.
 
 ### What is the criteria for a claimable referral?
 
-A claimable referral has the same definition as a Rewards eligible or Verified LBRY user. This means the referred user must have logged into the LBRY app and verified their account.  Users who redeemed rewards prior to the LBRY Open Beta (on 8/1/2017) were automatically grandfathered as verified users. Post LBRY Open Beta verification options can be seen [here](https://lbry.io/faq/identity-requirements). 
+A claimable referral has the same definition as a Rewards eligible or Verified LBRY user. This means the referred user must have logged into the LBRY app and verified their account.  Users who've redeemed rewards prior to the LBRY Open Beta (on 8/1/2017) were automatically "grandfathered" as verified users. Post-LBRY Open Beta verification options can be seen [here](https://lbry.io/faq/identity-requirements). 
```

Comment:
Not a big deal, but "Post-LBRY Open Beta verification options" sounds like LBRY is over. 
Perhaps "Current user verification options can be seen..."